### PR TITLE
update integration test to use new e2e suites

### PR DESCRIPTION
**Pull request by shorsher**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
+        test-suite: [TestEthereumMultipartyE2ESuite, TestEthereumGatewayE2ESuite, TestEthereumMultipartyTokensRemoteNameE2ESuite, TestEthereumNamespaceE2ESuite, TestFabricGatewayE2ESuite, TestFabricMultipartyE2ESuite]
```
Comment (shorsher) [inline .github/workflows/integration.yml:15]: Should we include the migration tests?

Comment (awrichar) [reply .github/workflows/integration.yml:15]: There's already a separate job (line 60)

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
...
         blockchain-provider: [geth, besu, fabric]
         database-type: [sqlite3, postgres]
         exclude:
```
Comment (awrichar) [inline .github/workflows/integration.yml:18]: The excluded combos need to be adjusted below for the new suite names. And you definitely need to exclude the combos of "fabric" with any test suite that uses tokens.

**Review by awrichar — APPROVED** [review]
Not sure of the best way to test this, but it LGTM.

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -12,16 +12,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
+        test-suite: [TestEthereumMultipartyE2ESuite, TestEthereumGatewayE2ESuite, TestEthereumMultipartyTokensRemoteNameE2ESuite, TestEthereumNamespaceE2ESuite, TestFabricGatewayE2ESuite, TestFabricMultipartyE2ESuite]
```
Comment (awrichar) [inline .github/workflows/integration.yml:15]: Actually can you pull any reference to `TestEthereumNamespaceE2ESuite`? It's just a subset of the main multiparty suite, and I think it's likely broken at the moment, and also probably not needed going forward due to the way identities work now.

**Pull request merged**